cmd/generator/main.go

```diff
@@ -120,28 +105,21 @@ var generatorCmd = &cobra.Command{
 		for _, repositoryName := range repositoryNames {
 			logger.Debugf("querying repository %s", repositoryName)
 
-			releasePlugins, err := getReleasePlugins(ctx, client, repositoryName, includePreRelease, existingPlugins)
+			var releasePlugins []*model.Plugin
+			releasePlugins, err = getReleasePlugins(ctx, client, repositoryName, includePreRelease, existingPlugins)
```

Comment:
nit: we can inline this if

Reply:
Unfortunately we can't, as `err` would shadow then.